src/components/form/form.scss

```diff
@@ -0,0 +1,25 @@
+.action-buttons {
+  padding: 10px 0;
+  display: flex;
+}
+
+.no-hover {
+  &:hover {
+    cursor: initial !important;
+    background-color: whitesmoke !important;
+  }
+}
```

Comment:
We can also remove this one when we start using [this](https://taskforce-fe-compone-git-fork-mmircea16-form-with-decisi-2be921.code4ro.now.sh/?path=/story/listitem--with-non-option-state) component